pkg/plugins/policies/meshcircuitbreaker/plugin/v1alpha1/plugin.go

```diff
@@ -161,6 +166,21 @@ func applyToGateways(
 					); err != nil {
 						return err
 					}
+
+					if dest.BackendRef == nil {
```

Comment:
I'm trying to understand, what's the case when it's nil

Reply:
added comment